org-formation/600-access/_tasks.yaml

```diff
@@ -430,3 +430,35 @@ SynapseLlmDeveloperPolicy:
         ]
       }
     PolicyName: SynapseLlmDeveloperPolicy
+
+SynapseLlmLambdaExecRoleBoundary:
```

Comment:
This is a policy boundry, so how about `SynapseLlmLambdaExecRolePolicyBoundary`?